### ISSUE TITLE
Not working on windows

### ISSUE THREAD

**Issue by sk-Prime** [mackwic/colored#59]
It is showing numbers rather than color. in this case `blue`
![Capture](https://user-images.githubusercontent.com/26684644/62410799-a92b3a00-b61d-11e9-9cd8-6a6adca31970.PNG)

Though i have added control::set_virtual_terminal(true); in my source file.
![Capture](https://user-images.githubusercontent.com/26684644/62410820-ea234e80-b61d-11e9-95dd-24712bdb73aa.PNG)

**Comment by kurtlawrence** [comment]
Hi @sk-Prime,

What version of Windows are you running? Which console?

I believe the virtual terminal sequence support is only available in Windows 10.

**Comment by sk-Prime** [comment]
in windows seven it doesn't work. but <a href="https://crates.io/crates/colour">colour</a> crate perfectly works. but i liked colored approach.

**Comment by lovasoa** [comment]
Yes, I can confirm it doesn't work on windows. Could this be mentioned in the README until it's fixed ?

**Comment by steelx** [comment]
im using Windows 10 with devmode on.
doesnt work on Windows Terminal neither on CMD.

**Comment by timvisee** [comment]
Same issue. Both in cmd and PowerShell. Fresh Windows 10 installation.

**Comment by kurtlawrence** [comment]
Hi all,

Could you please provide details about your OS, what shell/terminal you are using, an example of your code that does not work and a screenshot with the output showing it not working. Even the Rust toolchain you are using could be the issue.

I ask for this information as I am unable to reproduce on my machine, and in my experience with Rust production code, running on many machines (all Windows 10) that use `colored` there have been no issues when using the `set_virtual_terminal` flag.

**Comment by Timmmm** [comment]
It works for me in the `Command Prompt` and Powershell. Windows 10.0.19042.928.

![image](https://user-images.githubusercontent.com/376842/116826168-9810b380-ab8a-11eb-9f3f-f5277600aa32.png)

**Comment by constituent** [comment]
I have encountered the same issue. 
When I run "cargo run" while developing, it works perfectly, even with cmd.exe. However, with the compiled binary, whether debug or release mode, whether cmd or powershell, the control chars are displayed instead of color.

**Comment by constituent** [comment]
Oh I think I find the answer. As the doc of ansi_term suggests, On Windows 10, the application must enable ANSI support first: `let enabled = ansi_term::enable_ansi_support();`

**Comment by hseager** [comment]
This is working well for me:

```
fn main() {
    if cfg!(target_os = "windows") {
        control::set_virtual_terminal(true).unwrap();
    }

   ...
}
```
